src/main/Extensions/VSCode/VSCodeExtension.ts

```diff
@@ -280,10 +290,50 @@ export class VSCodeExtension implements Extension {
         };
     }
 
+    private instant: SearchResultItem[] = [];
+
+    private getFilesystemSearchResults(searchTerm: string, template: string): InstantSearchResultItems {
+        searchTerm = searchTerm.replace("~", (process.env as Record<string, string>).HOME);
+
+        if (this.fileSystemUtility.isDirectory(searchTerm)) {
+            const entries = this.fileSystemUtility.readDirectorySync(searchTerm, false);
+
+            this.instant = entries.map((e) => this.getFilePathItem(e, template, this.fileSystemUtility.isDirectory(e)));
+        }
+
+        return {
+            after: this.instant,
+            before: [],
+        };
+    }
+
+    private getFilePathItem(path: string, template: string, isDir: boolean): SearchResultItem {
+        return {
+            id: `vscode-filepath-${path}`,
+            name: path,
+            description: isDir ? "Folder" : "File",
+            image: isDir ? this.getImage() : this.getDefaultFileImage(),
```

Comment:
What do you think about showing only the filename/foldername in the `name` and adding the full path to the `details`?

Here is how that would look like with the "compact" search result list layout:
<img width="614" alt="image" src="https://github.com/user-attachments/assets/b01c3bad-8329-4280-bdec-a4a18a0698f5" />

And here with the new "detailed" search result list layout:
<img width="613" alt="image" src="https://github.com/user-attachments/assets/f9f77e9a-541e-421b-8a72-136b567fd25a" />

Reply:
Yeah that makes way more sense thanks